### ISSUE TITLE
Login from behind the proxy does not work

### ISSUE THREAD

**Issue by obrejla** [Microsoft/vscode-vsce#157]
Hi,

when I try to log in from behind the proxy, a timeout occurs.

```
dhcp-10-163-40-153:v4.5.0 ondrejbrejla$ vsce login obrejla
Personal Access Token for publisher 'obrejla': ****************************************************

Error: connect ETIMEDOUT 191.238.172.191:443
```

Thanks!

**Comment by joaomoreno** [comment]
Yeah proxies are not supported by VSCE for now. I suggest to publish from another machine/network.

**Comment by obrejla** [comment]
Yep, I workarounded that. Is it written somewhere? I didn't find any notice about proxies on [Publishing Tool](https://code.visualstudio.com/docs/tools/vscecli) age. It would be better to have it somewhere. Thanks!

**Comment by vikyd** [comment]
same issue

**Comment by Chazing** [comment]
same issue, cant publish using devOps from our corporate network.

**Comment by ShawnZhong** [comment]
Same issue here

**Comment by bkelley13** [comment]
Same.

**Comment by yasinkocak** [comment]
When will this solved?

**Comment by t2025** [comment]
Hi 
Any updates , when it will be solved ?

**Comment by t2025** [comment]
> Yep, I workarounded that. Is it written somewhere? I didn't find any notice about proxies on [Publishing Tool](https://code.visualstudio.com/docs/tools/vscecli) page. It would be better to have it somewhere. Thanks!

Can you please share your workaround? I really need solution.

Thanks

**Comment by JanChvala** [comment]
This also affects the **vsce pacakge** command which hangs on some requests. So the packaging process takes couple minutes instead of 10ths of seconds.

Is there any configuration that could prevent the packaging process to make any HTTP calls?
